python_modules/dagster/dagster/_seven/compat/pendulum.py

```diff
@@ -22,28 +22,31 @@ def mock_pendulum_timezone(override_timezone):
 
 
 def create_pendulum_time(year, month, day, *args, **kwargs):
```

Comment:
I feel a little weird about adding the tzinfo case here, just because this method was intended to be a relatively simple back-compat wrapper for the fact that pendulum.datetime got mapped to pendulum.create in pendulum 2 (as opposed to a one-stop shop for all pendulum time creation). Since tzinfo is not a valid argument to either pendulum.create or pendulum.datetime, would it be a big lift to move the tzinfo cases into the one callsite that does that?

Reply:
simplified this 👍